packages/bitcore-node/src/models/transaction.ts

```diff
@@ -428,14 +428,24 @@ export class TransactionModel extends BaseModel<ITransaction> {
 
     const walletConfig = Config.for('api').wallets;
     if (initialSyncComplete || (walletConfig && walletConfig.allowCreationBeforeCompleteSync)) {
-      let mintOpsAddresses = {};
-      for (const mintOp of mintOps) {
-        mintOpsAddresses[mintOp.updateOne.update.$set.address] = true;
-      }
-      let wallets = await WalletAddressStorage.collection
-        .find({ address: { $in: Object.keys(mintOpsAddresses) }, chain, network }, { batchSize: 100 })
-        .project({ wallet: 1, address: 1 })
-        .toArray();
+      let mintOpsAddressesSet = {};
+      for (const mintOp of mintOps) { mintOpsAddressesSet[mintOp.updateOne.update.$set.address] = true; }
+      var mintOpsAddresses = Object.keys(mintOpsAddressesSet);
```

Comment:
This looks fine, can you use prettier to format?

Reply:
Sure, should be pretty now :)